rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -371,12 +358,12 @@ error_code sceNpTrophyCreateContext(vm::ptr<u32> context, vm::cptr<SceNpCommunic
 
 	// generate trophy context name
 	std::string name;
-	sceNpTrophy.warning("sceNpTrophyCreateContext term=%s data=%s num=%d", commId->term, commId->data, commId->num);
+	sceNpTrophy.warning("sceNpTrophyCreateContext data=%s term=%s num=%d", commId->data, commId->term, commId->num);
```

Comment:
If you are already modifying this line:
```suggestion
	sceNpTrophy.warning("sceNpTrophyCreateContext(): data=%s term=%s (0x%x) num=%d", commId->data, commId->term, static_cast<u8>(commId->term), commId->num);
```
Allows to see the actual value of term.

Reply:
Actually you can drop both code paths and always use `std::sting_view(commId->data, std::sting_view(commId->data, 9).find_first_of('\0) + 1)` for name. The logic behind this is that it tries to for a null terminator in the 9 characters range and if it wont find it the string will only be 9 characters long.

Reply:
Doesn't really seem to work as expected.
![image](https://user-images.githubusercontent.com/23019877/93755267-4753d800-fc03-11ea-8099-11448507c913.png)

Reply:
ah right if find_first_of returns umax then replace with 9.

Reply:
std::string_view str(commId.data, 9);

if (auto pos = str.find_first_of('/'); pos != umax)
{
   str.remove_suffix(9 - pos);
}

Reply:
Should be better.

Reply:
Simplified further.